internal/pkg/api/container/container.go

```diff
@@ -404,6 +404,69 @@ func ContainerShow(csp *wwapiv1.ContainerShowParameter) (response *wwapiv1.Conta
 	return
 }
 
+func ContainerRename(crp *wwapiv1.ContainerRenameParameter) (err error) {
```

Comment:
I wish this wasn't being done as part of the api; but it does match the existing pattern, and I expect we hadn't talked about that yet when you did this.

Assuming this all works (I haven't actually tested it yet) I don't think we should refactor it right now; but this is one of those things I'll want to simplify in v4.6.0.